### DISCUSSION TITLE
Highlight differences between moonraker-timelapse and timelapse.py in the helper-script/files

### DISCUSSION THREAD

**Discussion by dhananjaipai** [Guilouz/Creality-Helper-Script-Wiki#613] [Issues]
I see that there are a few diffs between the timelapse.py in the [moonraker repo](https://github.com/mainsail-crew/moonraker-timelapse/blob/main/component/timelapse.py#L630) and the[ helper script repo](https://github.com/Guilouz/Creality-Helper-Script/blob/main/files/moonraker-timelapse/timelapse.py#L574)

The major one is how the filename format is modified from timelapse_ to k1_ which was confusing and had to spend some time to understand how to get the filename and how it was getting the new prefix!
Documenting this in the wiki would help, since in the wiki you are referring to the original repo for configurations but not using it for the actual python source, which can lead to inconsistent results like above and in the future.

But thanks again for your effort and appreciate all the help in making it easy for 3D printing noobs like me to get started!

**Comment by quiqueGCK1** [comment]
Speaking of timelapse in moonraker, until recently when the printing was finished, the timelapse generated two files, one with an mp4 extension and the other a jpg, (video and the previous one of the last frame), the names of these were the same as the name of the gcode, now and I don't know why, it only saves the mp4 and the name is a unique number without any apparent logic, do I have to configure something somewhere?
Thank you so much
